hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java

```diff
@@ -77,7 +77,7 @@
           EntityConstants.getValue(BackendAttribute.BACKEND_ATTRIBUTE_PATH),
           createAttributeValue(path));
     }
-    entityBuilder.putIdentifyingAttributes(RawSpanConstants.getValue(Sql.SQL_DB_TYPE),
+    entityBuilder.putAttributes(RawSpanConstants.getValue(Sql.SQL_DB_TYPE),
```

Comment:
Didn't this burn us a while ago in prod?

Reply:
CC @buchi-busireddy

Reply:
What do you mean?

Reply:
If this isn't an identifying attribute for BACKEND entity, it should be fine.

```
rs0:PRIMARY> db.entity_types.find({name:"BACKEND"}).pretty()
{
	"_id" : "__root:BACKEND",
	"_lastUpdateTime" : ISODate("2020-08-06T17:36:06.202Z"),
	"attributeType" : [
		{
			"name" : "BACKEND_HOST",
			"valueKind" : "TYPE_STRING",
			"identifyingAttribute" : true
		},
		{
			"name" : "BACKEND_PORT",
			"valueKind" : "TYPE_STRING",
			"identifyingAttribute" : true
		},
		{
			"name" : "BACKEND_PATH",
			"valueKind" : "TYPE_STRING"
		},
		{
			"name" : "BACKEND_PROTOCOL",
			"valueKind" : "TYPE_STRING",
			"identifyingAttribute" : true
		}
	],
	"createdTime" : NumberLong("1596656818940"),
	"name" : "BACKEND",
	"tenantId" : "__root"
}
```